assignments/assignment_3/README.md

```diff
@@ -2,31 +2,60 @@
 
-We currently have a single integration test, but that test is very dangerous: it deletes any files created by the application - including ones resulting from its normal usage. Ideally, we would either not have our tests change actual data and not depend on connectivity in order to work, or have a database made specifically for testing.
+We currently have a single integration test and this test was very dangerous: before you refactored the clean_data() function, it would overwrite any files created by the application - including ones resulting from its normal usage. Ideally, we would either not have our tests change actual data and not depend on connectivity in order to work, or have a database made specifically for testing. Therefore, let us solve this issue in this exercise.
 
 > Don't forget to create a branch for this assignment.
 
+## 0- VSCode Test Discovery
+
+In order to make your life easier during this assignment, we want to give you a little tip. If you're on VSCode, you can use the [Python Test Explorer](https://marketplace.visualstudio.com/items?itemName=LittleFoxTeam.vscode-python-test-adapter) extension to run your tests. It will automatically discover your tests and you can run them from the Test Explorer tab. At this point, if you have refactored your functions as requested in [assignment_2](../assignment_2/README.md) your tests should be failing.
+
 ## 1- Fixtures
 
-1. Create a fixture based on the `life_expectancy\data\eu_life_expectancy_raw.tsv` data. It can be a sample, so that our tests run faster. Then, replace the `life_expectancy\tests\fixtures\pt_life_expectancy_expected.csv` with the new expected data.
-2. Modify your `main` function so that the cleaned DataFrame is always returned. That will allow you to compare it with the expected fixture.
-3. Make the tests consume this fixture instead of the actual data. The idea of using fixtures instead of collecting data from a database will ensure our code is not dependent on connectivity. This will make our lives easier when we want to run our tests in a CI/CD pipeline.
+0. If you recall the structure from [assignment_0](../assignment_0/README.md), we currently have two fixtures, one for the expected output of the portuguese life expectancy dataframe and another one for the european life expectancy dataframe.
+1. Following best practises, we want to create a fixture that represents our data. The idea of using fixtures instead of collecting data from a database will ensure our code is not dependent on connectivity. This will make our lives easier when we want to run our tests in a CI/CD pipeline. Start from the current data in `life_expectancy\data\eu_life_expectancy_raw.tsv` and create a fixture `life_expectancy\tests\fixtures\eu_life_expectancy_raw.tsv` for the tests to consume. Since we only want to test our functions, this fixture does not need to be a copy of the original, but could rather be a smaller sample, so that our tests run faster.
```

Comment:
Maybe we can explain this within the assignment and stress the benefits of using a samples instead of all data. I think I would also divide the step into some substeps like:
 1.a) raw sample fixture
 1.b) expected sample fixture
 with some text in the middle of course. I'll prepare something in these lines.